Content.Server/Flash/FlashSystem.cs

```diff
@@ -63,7 +76,7 @@ public void OnUseInHand(EntityUid uid, FlashComponent comp, UseInHandEvent args)
 
             foreach (var entity in IoCManager.Resolve<IEntityLookup>().GetEntitiesInRange(comp.Owner.Transform.Coordinates, comp.Range))
             {
-                FlashEntity(entity, args.User, comp.AoeFlashDuration, comp.SlowTo);
+                Flash(entity.Uid, args.User.Uid, uid, comp.AoeFlashDuration, comp.SlowTo);
```

Comment:
Github doesn't let me select the line but this iterator is doing the resolve for IEntityLookup when it's already cached.

Reply:
Seems I forgot this one